test/xpu/test_conv.py

```diff
@@ -1240,6 +1241,30 @@ def test_conv_double_backward_strided_with_3D_input_and_weight(self, device):
         self.assertEqual(grad_input.shape, input.shape)
         self.assertEqual(grad_weight.shape, weight.shape)
 
+    def test_channels_last_ouput_stride(self, device):
+        dtype = torch.float32
+
+        def get_conv_out(device, dtype):
+            input = torch.randn(
+                (2, 3, 16, 16), device=device, dtype=dtype, requires_grad=True
+            )
+            weight = torch.randn(
+                (512, 3, 3, 3), device=device, dtype=dtype, requires_grad=True
+            )
+            input = input.to(memory_format=torch.channels_last)
+            weight = weight.to(memory_format=torch.channels_last)
+            out = torch.conv2d(input, weight, None, (2, 2), (0, 0), (1, 1), 1)
+            return out
+
+        # input NHWC, output NHWC
+        out = get_conv_out(device, torch.float32)
+        assert_size_stride(out, (2, 512, 7, 7), (25088, 1, 3584, 512))
+
+        # Like most conv backend, xpu does not support float64 for chanel last conv.
+        # input NHWC, output NCHW
+        out = get_conv_out(device, torch.float64)
+        assert_size_stride(out, (2, 512, 7, 7), (25088, 49, 7, 1))
```

Comment:
It is xpu-specific. Please separate it from current case and add a decorator - `@onXPUOnly`